docs/SuperSIDonLinux.md

```diff
@@ -102,11 +110,12 @@ If this Linux system is dedicated to SuperSid or you do not plan to mix various
 libraries: install at system level all the libraries.
 
 For an local installation inside the virtual environment, first execute 'workon

```

Comment:
For a local installation inside the venv virtual environment, execute 
source supersid-env/bin/activate

"workon" is for a different virtual environment package.

Reply:
To me it appears as if the description for 'workon' has been edited by you with https://github.com/sberl/supersid/commit/2d2a7aad55554c5923725ca07b99370745d93cf8#diff-b6d984c97cf29d4137300f67447b3957cdc78618281b4a918e2f6d7c87c568ba

See lines 104, 105.

I took, what I got. Just the inconsistency between 'supersid' and 'supersid-env' has been fixed by me. This was probably the point when you found a problem.

Let me know which wrong text to replace with which right text, please.

Reply:
I must have missed this when changed to use venv. It would be good to fix it now.